include/wx/generic/grid.h

```diff
@@ -1348,8 +1352,16 @@ class WXDLLIMPEXP_CORE wxGrid : public wxScrolledWindow
     void     SetColFormatBool(int col);
     void     SetColFormatNumber(int col);
     void     SetColFormatFloat(int col, int width = -1, int precision = -1);
+    void     SetColFormatDate(int col, const wxString& format = wxString());
     void     SetColFormatCustom(int col, const wxString& typeName);
 
+    // same for individual cells
```

Comment:
Not really, except the grid example. I've added these because they seem natural for me, I searched for such a function and was surprised it wasn't there. But now I see, that wxGrid API seems to be more column-oriented, and since you have concerns with these I will just remove them.